Cargo.toml

```diff
@@ -67,6 +71,8 @@ env_logger = "0.7"
 base64 = "^0.11"
 clap = "2.33"
 serial_test = "0.4"
+rstest = "^0.7"
+tiny-bip39 = "^0.7"
```

Comment:
The clippy error seems to be caused by this line, error goes away when I remove it. Could be because version `^0.8` is defined above (line 33) ?

Reply:
yes, you're right. Thanks for finding that.